src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -148,26 +148,34 @@ def create(input_dataset, target, feature=None, validation_set='auto',
     classes = sorted(classes)
     class_to_index = {name: index for index, name in enumerate(classes)}
 
+    validation_set_corrective_string = ("'validation_set' parameter must be "
```

Comment:
You should also be able to pass `None` if you want all of the data to be used for training.

Reply:
Look like you can do that. This error message makes it should like you can't.

Reply:
Done.